plugins/modules/server.py

```diff
@@ -471,9 +471,9 @@ def _create_server(self):
                 self.result["root_password"] = resp.root_password
                 # Action should take 60 to 90 seconds on average, but can be >10m when creating a
                 # server from a custom images
-                resp.action.wait_until_finished(max_retries=362)  # 362 retries >= 1802 seconds
+                resp.action.wait_until_finished(max_retries=1000)  # 362 retries >= 1802 seconds
```

Comment:
```suggestion
                resp.action.wait_until_finished(max_retries=362)  # 362 retries >= 1802 seconds
```

As far as I could tell the issue happens not with the `create_server` action that is being waited on here, but with the `resp.next_actions` 2 lines below.